spec/controllers/ops_controller/settings/upload_spec.rb

```diff
@@ -3,6 +3,166 @@
     stub_user(:features => :all)
   end
 
+  describe '#upload_png' do
+    let(:dir) { File.dirname(__FILE__) }
+
+    before do
+      allow(controller).to receive(:load_edit).and_return(true)
+      allow(controller).to receive(:redirect_to)
+      controller.instance_variable_set(:@sb, :active_tab => 'settings_custom_logos', :selected_server_id => 123)
+    end
+
+    it 'adds success flash message to @flash_array regarding updating custom logo image with a .png file' do
+      file = dir << '/data/test.png'
```

Comment:
So, like here, you can do

```suggestion
      file = File.join(__dir__, '/data/test.png')
```